### DISCUSSION TITLE
unable to verify school email for github campus expert

### DISCUSSION THREAD

**Discussion by fasihmuhammadvirk** [community/community#63133] [GitHub Education]
### Hi everyone,

I can not get my school email verified i have sent the verify and chek my email and junk and everything else but did not get any veridication code i want to apply for the campus expert program but i am unable too as it is not verify my shcool email.

**Comment by humairshoukat** [comment]
try submitting your Student ID Card picture from mobile. trying taking the picture while submitting the document.

**Reply by fasihmuhammadvirk** [reply]
can you please guide me like from where and what is the procedure i tried but i am not able to find how

**Reply by humairshoukat** [reply]
apply for student developer pack again and on the "verifying student status" step, submit the student card picture there.

**Reply by fasihmuhammadvirk** [reply]
![image](https://github.com/community/community/assets/82199241/248fb814-d5b8-49b4-bd6b-d588a61bc335)


This it what i get I can't go further

**Reply by fasihmuhammadvirk** [reply]
And i am not able to verify the email that my university gave me as i am not able to get any verification code on it

**Comment by fasihmuhammadvirk** [comment]
i can't proceed until i donot gave them my school email and i am not able to as it cannot be verify so i am stuck where they asked for the email and i canot clearly understand where you are refering me to sorry tp borther you this much but i am really confused

**Reply by humairshoukat** [reply]
try submitting your mail from the phone

**Reply by fasihmuhammadvirk** [reply]
ok i will try that

**Reply by fasihmuhammadvirk** [reply]
Can you tell me exactly where in github mobile to verify email and all I can't find it
